src/core/boot.c

```diff
@@ -19,71 +17,57 @@ void* find_multiboot_tag(multiboot_info_t* mbi, uint16_t type) {
 reserved_areas_t find_reserved_areas(multiboot_info_t* mbi) {
     multiboot_tag_t* tag;
     reserved_areas_t reserved = {
-        .kernel_start = -1,
-        .kernel_end = 0,
-        .multiboot_start = (uint64_t) mbi,
-        .multiboot_end = 0
+        .kernel_start = -1, .kernel_end = 0, .multiboot_start = (uint64_t)mbi, .multiboot_end = 0
```

Comment:
I prefer the multiline you had before, personally

Reply:
I like it better too but adopting existing conventions seems worthy